lib/src/cli.rs

```diff
@@ -929,7 +928,7 @@ async fn run_from_opt(opt: Opt) -> Result<()> {
             StateOpts::WipeOstree => {
                 let sysroot = ostree::Sysroot::new_default();
                 sysroot.load(gio::Cancellable::NONE)?;
-                wipe_ostree(&sysroot).await?;
+                tokio::task::spawn_blocking(move || crate::deploy::wipe_ostree(&sysroot)).await??;
```

Comment:
This is a super minor nit, stylistic but I would have put the spawn_blocking inside `wipe_ostree` instead of outside.

Basically it's good to keep `cli.rs` a relatively thin wrapper around "parse options, dispatch to function"

Reply:
Yeah I considered that, but saw a few calls to `spawn_blocking` in `cli.rs` which convinced me to keep it outside, but it was mostly arbitrary. Will move it inside

Reply:
Yep that's totally fine in this instance. However it's more common in some other code to *clone* the sysroot (it's effectively an `Arc<T>` internally (with gobject); we also promise that it's threadsafe, which is almost true).